Fix load errors

VictorHarbo opened the pull request:
Fixes load errors by adding a predefined api call as standard value for the textInput. 

Do you like this idea?

VictorHarbo committed:
Fix load errors

maxodsbjerg commented on URLDemonstrationAPP/app.R:
```diff
@@ -14,8 +14,9 @@ ui <- navbarPage(
-                           value = "")),
+                           p("Url:"),
+                           value = "http://labs.statsbiblioteket.dk/labsapi/api/aviser/export/fields?query=revolution%20AND%20py%3A%5B1840%20TO%201880%5D&fields=link&fields=recordID&fields=timestamp&fields=pwa&fields=cer&fields=fulltext_org&fields=pageUUID&fields=editionUUID&fields=titleUUID&fields=editionId&fields=familyId&fields=newspaper_page&fields=newspaper_edition&fields=lplace&fields=locat ...
+                           placeholder = "copy URL here")),
```
Even though this is a solution I think it might cause more confusion on where, what these little statistics are. I think a better idea would be adding an action button and then leaving the value of the textInput empty with the placeholder saying some like "Paste URL here". To read more about buttons in Shiny see [Actions Buttons](https://shiny.posit.co/r/articles/build/action-buttons/). _Pattern 2 - Delay reactions_ shows an example thats pretty close to what I imagined

maxodsbjerg reviewed (changes requested):
I suggest adding an action button. See more details in comment

VictorHarbo committed:
Add load button

VictorHarbo commented:
Ready for next review. Added a button, deleted some unused code and added a variable to store a boilerplate URL, which makes it easier to test the app.

maxodsbjerg reviewed (approved):
LGTM
